x-pack/plugins/enterprise_search/public/applications/app_search/components/error_connecting/error_connecting.tsx

```diff
@@ -19,11 +19,9 @@ export const ErrorConnecting: React.FC = () => {
       <SetPageChrome />
       <SendTelemetry action="error" metric="cannot_connect" />
 
-      <EuiPage restrictWidth>
-        <EuiPageContent hasBorder>
-          <ErrorStatePrompt />
-        </EuiPageContent>
-      </EuiPage>
+      <KibanaPageTemplate isEmptyState>
+        <ErrorStatePrompt />
+      </KibanaPageTemplate>
```

Comment:
@cchoas Just checking, this prompt is pretty tall - does it matter if it butts up against the kibana header & bottom of the page w/ no padding?

<img width="1323" alt="" src="https://user-images.githubusercontent.com/549407/123300513-80d9ac80-d4cf-11eb-9d12-5c897423ba27.png">

Reply:
Thanks for the catch @constancecchen , fixed!

Reply:
Wahoo, looks terrific! Including a screenshot for posterity

<img width="1409" alt="" src="https://user-images.githubusercontent.com/549407/123323429-547f5980-d4ea-11eb-9023-62d17ed5dea1.png">